Makefile

```diff
@@ -29,7 +29,7 @@ BINDIR		:= bin/$(OS)/$(ARCH)
 GCC			:= gcc
 CFLAGS		:=
 LDFLAGS		:=
-GOOPTS		:= -tags netgo
+GOOPTS		:=
 GO 			:= GOOS=$(OS) GOARCH=$(ARCH) GO15VENDOREXPERIMENT=1 go
```

Comment:
Since we change the file and we are Go 1.7+ only now, I think that `GO15VENDOREXPERIMENT=1` should go too.